src/client/components/UserBar/actions.js

```diff
@@ -0,0 +1,19 @@
+import { ENDPOINT_URL } from '../../common/variables';
+
+// Action types
+export const LOGOUT_USER = 'LOGOUT_USER';
+
+// Action creator
+export const unmountCurrentUser = () => ({ type: LOGOUT_USER });
+
+// Dispatchers
+export const logoutCurrentUser = () => dispatch =>
+  fetch(`${ENDPOINT_URL}/logout`, {
+    method: 'POST',
+    mode: 'no-cors',
+    credentials: 'include'
+  })
+    .then(() => dispatch(unmountCurrentUser()))
```

Comment:
This one shouldn't be required to be called. Probably all action could be removed (```unmountCurrentUser```)

Reply:
Maybe put there ```window.location.replace``` - if someone will use that action again, it will be ensured, that browser will be refreshed.